src/components/Dashboard/dashboard.less

```diff
@@ -348,6 +358,13 @@
     color: #ffffff;
     margin-top: 6px;
     display: inline-block;
+
+    &.listItemOutcome {
```

Comment:
Can we use BEM here? I know we decided to go for a different approach in the future, but for now we should stick to it:
`&__listItem`

Reply:
I disagree. The only reasons I can think of for sticking to an old approach are: we are going to change something in bulk or it is not super related and were are in a hurry/hotfix mode. 

Each day I have more doubts we will have time for refactoring so, seeing the project is "stabilized" it is perfectly fine to start/continue with the new CSS approach.